examples/config.kinesis.reference.hocon

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2014-2021 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
I think copyright should be either everywhere or nowhere in the configs? I prefer latter.